templates/z-stream.Dockerfile-args

```diff
@@ -0,0 +1 @@
+BUILDVERSION=1.8.2
```

Comment:
why 1.8.2 instead of 1.8.1?

Reply:
that's the next zstream (even if it's never released, that should still point to "next")